dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -24,3 +24,65 @@ properties:
         (INT1 or INT2) the drdy line is attached to. This property is not
         mandatory and if not present it defaults to 1 which is the
         configuration at power-up.
+
+    # tap and tap-tap configuration section
+    # All default values are selected to match the power-up values.
+    # tap and tap-tap events can be generated on INT1 only.
+
+    tap-mode:
+      type: int
+      required: false
+      default: 0
+      description: Tap mode. Default is power-up configuration.
+      enum:
+        - 0 # Only Single Tap
+        - 1 # Single and Double Tap
+
+    tap-threshold:
+      type: array
+      required: false
+      default: [0, 0, 0]
+      description: Tap X/Y/Z axes threshold. Default is power-up configuration.
```

Comment:
Please fix the multiline string for all of the properties being added in this patch also.